utils/build/docker/dotnet/parametric/ddtracer_version.Dockerfile

```diff
@@ -1,12 +1,11 @@
 
-FROM mcr.microsoft.com/dotnet/sdk:6.0
+FROM mcr.microsoft.com/dotnet/sdk:8.0
 
 RUN apt-get update && apt-get install dos2unix
-
 WORKDIR /app
 
-COPY utils/build/docker/dotnet/install_ddtrace.sh utils/build/docker/dotnet/query-versions.fsx binaries* /binaries/
+COPY utils/build/docker/dotnet/parametric/install_ddtrace.sh utils/build/docker/dotnet/parametric/query-versions.fsx binaries* /binaries/
```

Comment:
Is there a way to use the same script? If possible, I'd prefer maintain only one script